pandas/core/frame.py

```diff
@@ -2947,9 +2947,37 @@ def set_index(self, keys, drop=True, append=False, inplace=False,
 
         Examples
         --------
-        >>> indexed_df = df.set_index(['A', 'B'])
-        >>> indexed_df2 = df.set_index(['A', [0, 1, 2, 0, 1, 2]])
-        >>> indexed_df3 = df.set_index([[0, 1, 2, 0, 1, 2]])
```

Comment:
For this case, it would also be nice to add an example (so what this is doing is passing the actual index values, instead of a column name to set as the index)